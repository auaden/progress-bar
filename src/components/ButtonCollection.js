import React from 'react'
import { map } from 'lodash'
import { Button } from 'semantic-ui-react'
// import './styles/ButtonCollectionStyles.css'

export default ({ buttons }) => {
  const mappedButtons = map(buttons, (button) => <Button >{button}</Button>)
  return (
    <Button.Group>
      {mappedButtons}
    </Button.Group>
  )
}
