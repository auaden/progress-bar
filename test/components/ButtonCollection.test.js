import React from 'react'
import ButtonCollection from '../../src/components/ButtonCollection'

test('render buttons', () => {
  const wrapper = global.shallow(
    <ButtonCollection buttons={[20]} />
  )
  expect(wrapper).toMatchSnapshot()
})
