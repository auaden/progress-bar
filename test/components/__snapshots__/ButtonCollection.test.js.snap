// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render buttons 1`] = `
<ButtonGroup
  fluid={true}
>
  <Button
    as="button"
    compact={true}
    key="0"
    negative={false}
    onClick={[Function]}
    positive={true}
  >
    +20
  </Button>
</ButtonGroup>
`;
